### PR TITLE
update ownerinfo()

### PR DESCRIPTION
**Pull request by azemon**
Per comment in https://support.insight.ly/hc/en-us/community/posts/210133267-Sending-Email-via-API
update ownerinfo() to return the email dropbox address. Also fix bug so that ownerinfo() returns the correct contact ID.

### CODE REVIEW

File: insightly.py [hunk 1]
```diff
@@ -264,6 +264,8 @@ def __init__(self, apikey='', version='2.2', dev=None, gzip=True, debug=True, te
                     self.owner_id = u.get('USER_ID', None)
                     self.owner_name = u.get('FIRST_NAME','') + ' ' + u.get('LAST_NAME','')
                     self.contact_id = u.get('CONTACT_ID', None)
+                    self.email_dropbox = '%s-%s@mailbox.insight.ly' % (u.get('FIRST_NAME', '').lower(),
```
Comment (patrick-insightly) [inline insightly.py:273]: We added a new field to the api response object called EMAIL_DROPBOX_ADDRESS, should have it in production within a couple of days. If we could just map that new field to self.email_dropbox that would be great!

File: insightly.py [hunk 2]
```diff
@@ -912,7 +913,7 @@ def ownerinfo(self):
         """
         :return: dictionary of information about the account owner
         """
-        return {'name': self.owner_name, 'email': self.owner_email, 'id': self.owner_id}
```
Comment (patrick-insightly) [inline insightly.py:922]: There may be cases where owner_id may be useful to have as part of the ownerinfo, could we please keep it in addition to contact_id?

File: insightly.py (hunk 1 above)
Comment (azemon) [inline insightly.py:273]: I agree. Using EMAIL_DROPBOX_ADDRESS is much better than what I proposed here.

File: insightly.py (hunk 2 above)
Comment (azemon) [inline insightly.py:922]: I'm easy either way. I was the one who added owner_id (it was my mistake).

Comment (patrick-insightly) [inline insightly.py:922]: Cool, let's keep both then!

File: insightly.py (hunk 1 above)
Comment (patrick-insightly) [inline insightly.py:273]: Thanks Art!

**Pull request merged**